packages/packagers/js/src/ScopeHoistingPackager.js

```diff
@@ -834,7 +834,10 @@ ${code}
       // use a helper to check the __esModule flag at runtime.
       let kind = dep?.meta.kind;
       if (
-        (!dep || kind === 'Import' || kind === 'Export') &&
+        (!dep ||
+          kind === 'Import' ||
+          kind === 'Require' ||
```

Comment:
Adding this condition fixes this particular case but breaks the case where we should not insert a default interop for wrapped CJS modules. What should we do here? @mischnic